app/models/solidus_avatax_certified/line.rb

```diff
@@ -150,6 +156,30 @@ def base_line_hash
       }
     end
 
+    def tax_override_hash
+      return {} unless @override_tax
```

Comment:
Do we need this? Wouldn't it go to the `else` case anyway in this scenario?

Reply:
ya this is not needed